logbook-api/src/main/java/org/zalando/logbook/BaseHttpMessage.java

```diff
@@ -43,7 +43,7 @@
 
     class HeadersBuilder {
 
-        private final Map<String, List<String>> headers;
+        private Map<String, List<String>> headers;
```

Comment:
I think builders should be reusable, otherwise we are violating the Principle of least astonishment.